src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,77 @@
+package duke.command;
+
+import duke.task.Task;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import static duke.command.Command.*;
+
+import java.util.ArrayList;
+
+public class Duke {
+    public static void main(String[] args) {
+
+        printWelcomeMessage();
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        try {
+            readFile("data/duke.txt", tasks);
+        } catch (IOException e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+        }
+
+        untilBye:
+        while (true) {
+            try {
+                String line = in.nextLine();
+                String[] parsedInput = line.split(" ");
+                switch (parsedInput[0]) {
+                case "list":
+                    printTaskList(tasks);
+                    break;
+                case "mark":
+                    tryMarkTask(tasks, line);
+                    break;
+                case "unmark":
+                    tryUnmarkTask(tasks, line);
+                    break;
+                case "todo":
+                    tryAddTodo(tasks, line);
+                    break;
+                case "deadline":
+                    tryAddDeadline(tasks, line);
+                    break;
+                case "event":
+                    tryAddEvent(tasks, line);
+                    break;
+                case "bye":
+                    printByeMessage();
+                    break untilBye;
+                case "delete":
+                    tryDeleteTask(tasks, parsedInput);
+                    break;
+                default:
+                    throw new DukeException();
+                    // Fallthrough
```

Comment:
Can remove this comment if its meaning is obvious from the code. 

Code Quality Guideline: Comment minimally but sufficiently